docs.html

```diff
@@ -644,6 +652,92 @@ <h3 class="card-title">Our Chaning Work</h3>
+    <div class="drawer-links">
+      <i class="fas fa-star drawer-icon"></i>
+      Star
+    </div>
+    <div class="drawer-links">
+      <i class="fas fa-paper-plane drawer-icon"></i>
+      Sent Mail
+    </div>
+    <div class="drawer-links">
+      <i class="fas fa-envelope-open drawer-icon"></i>
+      Drafts
+    </div>
+  </div>
+</div>
+
+<div class="drawer">
+  <h2 class="drawer-title center">Title</h2>
+  <p class="drawer-subtext center">subtext</p>
+  <div class="drawer-nav">
+    <div class="drawer-side-links">
+      Inbox
+      <i class="fas fa-archive drawer-icon"></i>
+    </div>
+    <div class="drawer-side-links">
+      Star
+      <i class="fas fa-star drawer-icon"></i>
+    </div>
+    <div class="drawer-side-links">
+      Sent Mail
+      <i class="fas fa-paper-plane drawer-icon"></i>
+    </div>
+    <div class="drawer-side-links">
+      Drafts
+      <i class="fas fa-envelope-open drawer-icon"></i>
+    </div>
+  </div>
+</div>
+</div>
+
+
+<code>
+  <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=662&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%250A%253Cdiv%2520class%253D%2522drawer%2522%253E%250A%2520%2520%253Ch2%2520class%253D%2522drawer-title%2522%253ETitle%253C%252Fh2%253E%250A%2520%2520%253Cp%2520class%253D%2522drawer-subtext%2522%253Esubtext%253C%252Fp%253E%250A%2520%2520%253Cdiv%2520class%253D%2522drawer-nav%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522drawer-links%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-archive%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520Inbox%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522drawer-links%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-star%2520drawer-icon%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520Star%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522drawer-links%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-paper-plane%2520drawer-icon%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520Sent%2520Mail%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522drawer-links%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-envelope-open%2520drawer-icon%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520Drafts%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%252F%252Ffor%2520side-drawer%2520add%2520on%2520line%252010%2520%252C%252014%2520%252C%252018%2520%253B%250A%252F%252Fclass%253D%2522drawer-side-links%2522%2520%250A"
+  style="width: 100%; height: 400px; border:0; transform: scale(1); overflow:hidden;margin-top: 5px;margin-bottom: 50px"
```

Comment:
Looks good . You can add the style in a class and add to iframe instead of using inline styles . Also Install prettier if you haven't

Reply:
okay sure